cmd/api/generic.go

```diff
@@ -148,6 +149,16 @@ func makeTokenRoute(router gin.IRouter, api blockatlas.Platform) {
 	})
 }
 
+func makeCoinRoute(router gin.IRouter) {
+	router.GET("/coins", func(c *gin.Context) {
+		coins := make([]coin.Coin, 0)
+		for _, coin := range coin.Coins {
+			coins = append(coins, coin)
+		}
+		c.JSON(http.StatusOK, coins)
```

Comment:
is it possible just to return `c.JSON(http.StatusOK, []coin.Coin)` ?

Reply:
We can save the objects inside a slice in the coin package, to avoid the iteration every time we call this API. The structure inside a `map[uint]Coin` it is harder to parse